bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -216,43 +216,22 @@ export class PlotCanvasView extends DOMView
     follow_enabled = false
     has_bounds = false
 
+    r = null
```

Comment:
@mattpap  is there a better way to subsequently communicate this optional value to `update` ?

Reply:
No, this is idiomatic javascript.